pkg/asset/cluster/tfvars.go

```diff
@@ -338,16 +339,29 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		}
 		preexistingnetwork := installConfig.Config.GCP.Network != ""
 
-		imageRaw, err := rhcospkg.GCPRaw(ctx, installConfig.Config.ControlPlane.Architecture)
+		archName := coreosarch.RpmArch(fmt.Sprintf("%s", installConfig.Config.ControlPlane.Architecture))
+		st, err := rhcospkg.FetchCoreOSBuild(ctx)
 		if err != nil {
-			return errors.Wrap(err, "failed to find Raw GCP image URL")
+			return err
+		}
+		starch, err := st.GetArchitecture(archName)
+		if err != nil {
+			return err
+		}
+
+		var gcpimage string
+		if starch.Images.Gcp != nil {
+			img := starch.Images.Gcp
+			gcpimage = fmt.Sprintf("projects/%s/global/images/%s", img.Project, img.Name)
+		} else {
+			return fmt.Errorf("%s: No GCP build found", st.FormatPrefix(archName))
 		}
 		data, err := gcptfvars.TFVars(
 			gcptfvars.TFVarsSources{
 				Auth:               auth,
 				MasterConfigs:      masterConfigs,
 				WorkerConfigs:      workerConfigs,
-				ImageURI:           imageRaw,
+				ImageURI:           gcpimage,
```

Comment:
Is there any difference between this image URI and the one from the rhcos image asset any more? This code used to get the "raw image" but now gets the same image as the rhcos image asset does. Is that intentional?

Reply:
OK right, I should have looked at this more closely.  Some quick digging with git annotate points to
https://github.com/openshift/installer/commit/1f2d071f07ffd9e12fb9df64faadc3627db55bbd
which is related to the special licenses dance.  But in fact, we enable the nested virt license on the public RHCOS image by default since 4.6, which was the main motivation to support license overrides introduced in https://github.com/openshift/installer/pull/3430

So...we could do a prep commit which cleans this up and drops the support from the installer for overriding licenses?

Reply:
(Random aside, bits of CoreOS CI testing actually relies on a nested KVM setup in our own Prow instance)

Reply:
A prep commit would be nice so as not to lose the nuance of this change.

I am not very familiar with how the licenses work. Are you claiming that we do not need to support user-supplied licenses any more? If we switch away from using the raw image, we will lose support for user-supplied licenses?

Reply:
https://github.com/openshift/installer/pull/4696

Reply:
That is what I was afraid of. We cannot remove support for licenses. It is a breaking change.

Reply:
> Are you claiming that we do not need to support user-supplied licenses any more?

I am 93.2% sure there is no use case for other licenses beyond `enable-vmx` (which again is the default now), I looked and couldn't even find references to this.  I tried out the "license list" API linked there:
https://cloud.google.com/compute/docs/reference/rest/v1/licenses/list?apix_params=%7B%22project%22%3A%22openshift-gce-devel%22%2C%22returnPartialSuccess%22%3Atrue%7D
and I just got the empty set.

Reply:
OK I added a hack for this.  I'd still argue we can drop the license stuff, but if we decide it all needs to stay as is I can try plumbing through the image URL into the stream metadata.